keymanagementscenarios.md

```diff
@@ -0,0 +1,187 @@
+- Repository Owner: In some scenarios a container may be stored in a repository that is managed by the repository owner. The repository may be public or private and could also be air-gapped.
+
+## Definitions:
+- Root key: A self signed key used for the lowest designation of trust. Root keys can be created by developers, organizations, public/private CAs, and registry operators. The root key should be retrieved from a trusted source that can establish the authenticity of the creator's identity.
+- Signing key: A signing key is used to generate artifact signatures. A signing key should be signed with a root key or one of its intermediaries. The certificate chain with a signing key can be used to verify which root it belongs to. While a root key can be used as a signing key, this is not recommended as it creates a large blast radius and increases the risk of compromising a root key. 
+- Trust Store: The trust store defines the relationship between signing keys and artifacts that are used at validation time to determine whether to trust an artifact with a crytptographically valid signature. The trust store will relate a source (any source, specific registry, specific repository, or specific target) with a certificate (for root key, intermediate, or signing key) or key repository (for automated key distribtuion). It is not recommended to use a signing key as this will cause signature validation to fail if the signing key is rotated. 
+- Trust Store Key Information: The key information configured in the trust store will be cryptographically verifiable and contain:
+    - Public Key: Will be used to verify signed artifacts.
+    - Signed Identity: Identity of creator of the signing key.
+    - (Optional) Issuer: If the key is an intermediate this will describe the owner of the root.
+    - Mechanism to update validity of signatures generated with the signing key.
+
+## Requirements
+- Signing an artifact SHOULD NOT require the publisher to perform additional actions with a registry/repository or registry operator beyond those required to push an unsigned artifact.
+- Validating a signature SHOULD NOT require the deployer to perform additional actions with a registry/repository or registry operator beyond those required to pull an unsigned artifact.
+- Moving an artifact from one repository to another SHOULD NOT invalidate the signature on the artifact.
+- Publishers SHOULD be able to sign with keys stored on their local machines, secure tokens, Hardware Security Modules (HSMs), or cloud based Key Management Services.
+- Publishers SHOULD be able to generate multiple signatures for a single artifact.
+- Publishers MUST have a mechanism to revoke signatures to indicate they are no longer trusted. 
+- Trust stores SHOULD be configurable by the deployer.
+- Deployers MUST be able to configure trusted entities for individual repositories and targets.
```

Comment:
There is a missing subtlety here around how trusted entities and trust stores interact. What you're describing is pinning to one or more identities (i.e. you don't want to deploy everything that chains to a root in the store, but what you do trust, MUST chain to a root in the trust store). 

This implies that the owner of the root and/or their delegates MUST manage identities somehow.